apps/frontend/src/app/app.routes.ts

```diff
@@ -1,11 +1,11 @@
 import { Route } from '@angular/router';
 
-import { AppRootLayout } from './layouts/root.layout';
+import { HeaderComponent } from './components/header/header.component';
 
 export const APP_ROUTES: Route[] = [
 	{
 		path: '',
-		component: AppRootLayout,
+		component: HeaderComponent,
```

Comment:
Weil man für Siemens iX außenrum <ix-application> braucht und dann direkt eine Ebene darunter sowas wie <ix header> oder .... haben wir es nur gelöst bekommen indem wir einfach alles in den Header geschrieben haben und die Root Seite sozusagen aufgegeben haben

Reply:
Dann geht das so nicht. Dann muss alles in den Header, ein Layout is schlicht etwas anderes  wie eine Component.

Reply:
okay I deleted the root layout then